internal/weldr/json.go

```diff
@@ -139,6 +139,7 @@ type SourceConfigV0 struct {
 	System   bool     `json:"system" toml:"system"`
 	Proxy    string   `json:"proxy,omitempty" toml:"proxy,omitempty"`
 	GPGUrls  []string `json:"gpgkey_urls,omitempty" toml:"gpgkey_urls,omitempty"`
+	RHSM     bool     `json:"rhsm" toml:"rhsm"`
```

Comment:
Technically this should only be added to the V1 Source structs/json since V0 is 'finished'.

Reply:
Where is V0 used?

Reply:
/api/v0/projects/source/* uses it.

Reply:
Sure, but how does one use it with composer-cli? I tried it on RHEL and it uses the V1 version.

Anyhow, I've made the changes you requested. Is it ok now?

Reply:
composer-cli uses the v1 route, but the v0 routes and responses should be left alone since they are 'done'.